IMPLEMENTATION_PLAN_tts_adjacent_defaults_and_voice_assistant.md

```
@@ -0,0 +1,23 @@
+## Stage 1: Lock Regressions With Tests
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)